experimental/addons/kruise-rollout/definitions/kruise-rollout.cue

```diff
@@ -0,0 +1,121 @@
+"kruise-rollout": {
+	type: "trait"
+	annotations: {}
+	labels: {}
+	description: "Rollout workload by kruise controller."
+	attributes: {
+		podDisruptive: true
+		appliesToWorkloads: ["*"]
```

Comment:
Please set the appliesToWorkloads, does the rollout only support the Deployment and CloneSet?
```
appliesToWorkloads: ["deployments.apps", "clonesets.apps.kruise.io"]
```